pkgs/tools/misc/file/default.nix

```diff
@@ -17,6 +17,13 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-N1HH+6jbyDHLjXzIr/IQNUWbjOUVXviwiAon0ChHXzs=";
   };
 
+  patches = [
+    # Backport fix to identification for pyzip files.
+    # Needed for strip-nondeterminism.
+    # https://salsa.debian.org/reproducible-builds/strip-nondeterminism/-/issues/20
+    ./pyzip.patch
```

Comment:
Maybe pull it from here:

https://github.com/file/file/commit/dc71304b3b1fd2ed5f7098d59fb7f6ef10cfdc85

Reply:
`fetchpatch` is usually hard to do during stdenv bootstrapping (or sometimes just costly).